3574/1/top_gems.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'parse'
+require_relative 'option_parser'
+require_relative 'read_file'
+require_relative 'table'
+require_relative 'option_functions'
+
+HEADER = %i[name_id used_by watch stars fork issues contributors].freeze
+
+class TopGems
+  def initialize
+    @gems_from_file = {}
+    @gems_from_file[:file_name] = 'list_gems.yml'
```

Comment:
for next 3 lines: 
@gems_from_file = {
  file_name: 'list_gems.yml',
  number: []
}

Reply:
Also it is not clear what is `number` and why it is an array

Reply:
And please move 'list_gems.yml' at least to constant. Or you can take it from env variables. You can refer `dotenv` gem if you wich